File: file_format_homework/filereader.js
const fs = require('fs');
const yaml = require('js-yaml');
try{
    const txtData = fs.readFileSync('birds.txt','utf8');
    console.log("TXT");
    console.log(txtData);
    const jsonData = JSON.parse(fs.readFileSync('birds.json','utf8'));
    console.log("JSON");
    console.log(jsonData);
    console.log('yaml')
    const yamlData = yaml.load(fs.readFileSync('birds.yaml','utf8'));
    console.log(yamlData)
    console.log('xml')
    const xmlData = fs.readFileSync('birds.xml','utf8')
    console.log(xmlData)
    console.log('csv')
    const csvData = fs.readFileSync('birds.csv','utf8')
    console.log(csvData)
} catch(e){
    console.log(e);
}

File: file_format_homework/filereader.py
import json
import yaml
txtData = open('birds.txt','r')
print('TXT')
print(txtData.read())
txtData.close()
jsonData = json.load(open('birds.json','r'))
print('JSON')
print(jsonData)
with open('birds.yaml','r') as stream:
    try:
        print('YAML')
        print(yaml.safe_load(stream))
    except yaml.YAMLError as exc:
        print(exc)
import csv
with open('birds.csv', newline='') as csvfile:
    spamreader = csv.reader(csvfile, delimiter=' ', quotechar='|')
    print('CSV')
    for row in spamreader:
        print(', '.join(row))
from bs4 import BeautifulSoup
with open('birds.xml', 'r') as f:
    data = f.read()
    Bs_data = BeautifulSoup(data, "xml")
    print('XML')
    print(Bs_data)